static/css/style.css

```diff
@@ -227,3 +227,15 @@ main {
   -moz-column-gap: 40px;
   column-gap: 40px;
 }
+
+h1, .h1 {
+  font-size: 2.14375rem; }
```

Comment:
Doesn't bootstrap already have this styles for h1, h2, ...?

Reply:
They are slightly different. I downsized them a bit because we used a lot of h3, h4, h5. So the old h3 is now h2...